src/main/scala/chiseltest/simulator/VerilatorSimulator.scala

```diff
@@ -234,7 +234,9 @@ private trait VerilatorSimulatorTrait extends Simulator {
     if (verbose) {
       // print the command and pipe the output to stdout
       println(cmd.mkString(" "))
-      println(makeScriptFromCommand(cmd, Some(cwd / s"chiseltest_command_${cmd.head}.sh")))
+      if (cwd != null) {
+        println(makeScriptFromCommand(cmd, Some(cwd / s"chiseltest_command_${cmd.head}.sh")))
```

Comment:
For you purposes, `cwd == null` essentially means `cwd == os.pwd`. So if you replace the `null` with `os.pwd` things should continue to work and you won't have to add the check here.